[TensorPipe Agent] Message on Agent Shutdown

osalpekar opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#38819 [TensorPipe Agent] Message on Agent Shutdown**

Logs a message when the agent is shutting down like the other RPC
Agents.

Differential Revision: [D21671061](https://our.internmc.facebook.com/intern/diff/D21671061/)

osalpekar committed:
[TensorPipe Agent] Message on Agent Shutdown

Logs a message when the agent is shutting down like the other RPC
Agents.

Differential Revision: [D21671061](https://our.internmc.facebook.com/intern/diff/D21671061/)

[ghstack-poisoned]

lw commented on torch/csrc/distributed/rpc/tensorpipe_agent.cpp:
```diff
@@ -487,6 +487,8 @@ void TensorPipeAgent::join() {
 
 void TensorPipeAgent::shutdownImpl() {
+  LOG(INFO) << "Shutting down TensorPipeAgent on node "
+            << RpcAgent::getWorkerInfo().id_;
```
Can we use the worker name instead of the rank? It's typically more informative

osalpekar committed:
Update on "[TensorPipe Agent] Message on Agent Shutdown"


Logs a message when the agent is shutting down like the other RPC
Agents.

Differential Revision: [D21671061](https://our.internmc.facebook.com/intern/diff/D21671061/)

[ghstack-poisoned]
